src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -267,9 +267,10 @@ private void configureCountryImageView() {
 
     countryImageView.setVisible(true);
 
-    if (!StringUtils.isEmpty(player.getCountry())) {
-      final Tooltip countryTooltip = new Tooltip(i18n.getCountryNameLocalized(player.getCountry()));
-      Tooltip.install(countryImageView, countryTooltip);
+    if (!StringUtils.isEmpty(player.getCountry())) { // empty (non-null) string -> empty tooltip
+      i18n.getCountryNameLocalized(player.getCountry()).ifPresent(countryName -> //
```

Comment:
Remove empty comment

Reply:
That is there to force a line break at this place.